app/pods/components/left-side-bar/template.hbs

```diff
@@ -7,7 +7,7 @@
         {{#if model.photo}}
         <img src="{{model.photo}}" class="img-resize img-circle" alt="">
        {{else}}
-        <img src="https://robohash.org/{{md5 model.name}}/?set=set4" class="img-resize img-circle"
+        <img src="https://minio.cb.lk/img/avatar-{{get-rand 36}}.svg" class="img-resize img-circle"
```

Comment:
Instead save this URL in our database so that the user doesn't see a new avatar every time he tries to fetch the leaderboard.